src/contentScripts/injected.ts

```diff
@@ -185,12 +184,10 @@ const off = (eventName: string, cb: (data: unknown) => void) => {
   EVENTS[eventName] = bucket.filter((callback) => callback === cb);
 };
 
-const emit = (eventName: string, payload?: any) => {
+const emit = (eventName: string, payload?: unknown) => {
```

Comment:
Are we sure that the payload is `unknown`?

Reply:
I think it's the same because we can emit any payload.